src/components/forms/dashboard/links/link-edit-form.tsx

```diff
@@ -110,10 +167,19 @@ const LinkEditForm = () => {
 
       <section className="grid grid-cols-1 gap-5 mt-6 md:grid-cols-11">
         <div className="flex flex-col col-span-5 gap-4">
+
           <div className="flex flex-col gap-1">
-            <h1 className="text-2xl">Create your Link</h1>
+            <div className="options" style={{display:"flex",flexDirection:"row",justifyContent:"space-between"}}>
+            <h1 className="text-2xl">Create your Link </h1>
+            <h1 className="text-2xl">Or</h1>
+            <button className="border border-white-900 border-2 px-3 py-2" onClick={()=>{
+              document.getElementById('inputCSV')?.click();
+            }}> Upload CSV</button>
```

Comment:
Let's use the `Button` component from the shadcn library we are using so we keep things consistent